src/org.hdfgroup.hdfview/hdf/view/TableView/DefaultCompoundDSTableView.java

```diff
@@ -183,15 +196,16 @@ protected NatTable createTable(Composite parent, DataFormat dataObject) {
         natTable.addLayerListener(new CompoundDSCellSelectionListener());
 
         // Create popup menu for region or object ref.
-        //if (isStdRef || isRegRef || isObjRef) {
-        //    natTable.addConfiguration(new RefContextMenu(natTable));
-        //}
+        natTable.addConfiguration(new RefContextMenu(natTable));
```

Comment:
Shouldn't the menu only be added when it's a reference type?

Reply:
The trouble is that I can't get the datatype for the field at this point, so I just enabled here for all fields. I believe the actual display is checked at selection time.

Reply:
Shouldn't all the fields be the same type? They might be a different reference type, but either the dataset is a reference datatype or it isn't.

Reply:
No not for compounds. The members could be different types.

Reply:
For compounds we should first determine if any of the fields are reference types before adding the context menu, but I suppose it won't do much harm anyway.

Reply:
That's why I left the if line commented.  TODO later.